salt_lsp/utils.py

```diff
@@ -83,7 +83,7 @@ def construct_path_to_position(tree: Tree, pos: Position) -> List[AstNode]:
     parser_pos = parser.Position(line=pos.line, col=pos.character)
 
     def visitor(node: AstNode) -> bool:
-        if parser_pos >= node.start and parser_pos < node.end:
+        if node.start <= parser_pos < node.end:
```

Comment:
Are you sure about this one? Iirc some programming languages don't actually give you what you expect in these double comparisons

Reply:
In Python this works, see https://note.nkmk.me/en/python-chain-comparisons/ (links to official docs as well)